nix/tests.nix

```diff
@@ -81,9 +81,9 @@ let
         rootdir="$(${git}/bin/git rev-parse --show-toplevel)"
         cd "$rootdir"
 
-        export PATH="${postgrestStatic}/bin:${curl}/bin:${ncat}/bin:$PATH"
+        export PATH="${curl}/bin:$PATH"
 
-        ${withTmpDb postgresql} "$rootdir"/test/io-tests.sh
+        ${withTmpDb postgresql} ${cabal-install}/bin/cabal v2-exec -f FailOnWarn --test-show-detail=direct "$rootdir"/test/io-tests.sh
```

Comment:
Lets use the `${devCabalOptions}` here!

Reply:
Oh boy... I just changed that before pushing, but forgot to commit it! :D